Mathlib/Topology/LocallyFinsupp.lean

```diff
@@ -343,6 +343,33 @@ Functions with locally finite support within `U` form an ordered commutative gro
 instance : IsOrderedAddMonoid (locallyFinsuppWithin U Y) where
   add_le_add_left := fun _ _ _ _ ↦ by simpa [le_def]
 
+/--
+The positive part of a sum is less than or equal to the sum of the positive parts.
+-/
+theorem posPart_add (f₁ f₂ : Function.locallyFinsuppWithin U Y) :
+    (f₁ + f₂)⁺ ≤ f₁⁺ + f₂⁺ := by
+  rw [instPosPart, Function.locallyFinsuppWithin.le_def]
```

Comment:
`rw [instPosPart, ...` suggests we're missing a lemma, or else you didn't use the one we have. Please add the lemma you need to perform this rewrite.